config.example.toml

```diff
@@ -1,3 +1,8 @@
+# Include any additional configuration file(s)
+# Note that this config file overrides the includes
+# and includes that are on the left override the ones on the right in this array
+#include = ["/etc/topgrade.toml"]
```

Comment:
Couldn't you use the glob crate to match an entire directory and create a directory dedicated to topgrade configurations, such as `/etc/topgrade.d/`?

Reply:
The linked issue to this pr asked to include files that are in this specific array, whether the directory they are located.

However I could match any file in `$XDG_CONFIG_HOME/topgrade/*` by default, but I would have to ask maintainers if that's a good idea

Reply:
Matching any file would be cool

Reply:
> Matching any file would be cool

I will push a commit to do that

Reply:
I think putting them at the beginning of the file (lowest priority) should be fine.

Reply:
Solved in [33d1fc6](https://github.com/topgrade-rs/topgrade/pull/421/commits/33d1fc6e6ea10bdd3ba53b06c9767720f8f95a33)